packages/mjml-carousel/src/CarouselImage.js

```diff
@@ -62,14 +62,17 @@ export default class MjCarouselImage extends BodyComponent {
   renderThumbnail() {
     const { carouselId, src, alt, 'tb-width': width, target } = this.attributes
     const imgIndex = this.props.index + 1
+    const cssClass = this.getAttribute('css-class')
+                   ? `${this.getAttribute('css-class')}-thumbnail`
```

Comment:
You can use multiple css-class on attribute, we already have this logic somewhere to prefix css classes with outlook. Might need to do an helper for that on core ?